### DISCUSSION TITLE
Configuring and Running first application

### DISCUSSION THREAD

**Discussion by sqageek** [dipjyotimetia/HybridTestFramework#414] [Q&A] [answered]
Hi,
I am new to all the technologies/frameworks used in this project. In the documentation, there is a step to add Environmental variables. But there is no "bin" folder in "allure2-2.17.2". 
Secondly, I cloned the repo into my local machine. Then navigate to that folder. Then, while importing the project I am getting the below error.
![image](https://user-images.githubusercontent.com/6234893/154958744-e441e998-e04a-4724-8057-590b0f25f0a2.png)

Please suggest to me how to solve the above issue? Are there any blog/video with more detailed information

**Comment by dipjyotimetia** [comment, marked as answer]
Please find the documentaion here
![image](https://user-images.githubusercontent.com/18288720/155138442-639b4e1d-2b12-4638-be37-66c328c13edf.png)
